Client/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -710,7 +710,6 @@ class NewTabPageViewController: UIViewController {
     if !feedDataSource.shouldLoadContent {
       return
     }
-    rewards.ads.purgeOrphanedAdEvents(.inlineContentAd) { _ in }
```

Comment:
Is it related to news? or just a cleanup on your part

Reply:
Related to news (inline content ads are the news ads), purge happens before first ad generation now